src/platformer/platformer.py

```diff
@@ -3,41 +3,53 @@
 Github repo can be found here:
 https://github.com/sparshg/pycollab
 """
+
+# Import statements
+import pygame as pg
 from os import environ
 
 # Hide pygame Hello prompt
 environ["PYGAME_HIDE_SUPPORT_PROMPT"] = "1"
```

Comment:
```suggestion

from os import environ

# Hide pygame Hello prompt
environ["PYGAME_HIDE_SUPPORT_PROMPT"] = "1"
import pygame as pg
```
pygame should be imported after hiding the prompt or it won't hide that

Reply:
Ah, got it, thanks.

Reply:
soo, merge?

Reply:
OK.
But can't we perhaps continue coding this when not so busy?

Reply:
ok, you can message me when you get free from exams so we can continue

Reply:
Wow, cool site! :D